mtlpy/templates/base.html

```diff
@@ -118,15 +107,19 @@ <h1 class="page-title">{% block title %}{% endblock %}</h1>
         <script>jQuery.fn.scrollspy || document.write("<script src='{% static 'js/bootstrap.min.js' %}'><\/script>");</script>
 
         {% block extra_script %}{% endblock %}
-	  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-	  <script type="text/javascript">
-	    window.___gcfg = {lang: '{% if LANGUAGE_CODE == 'fr' %}fr-CA{% else %}en-US{% endif %}'};
-
-	    (function() {
-	    var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
-	    po.src = 'https://apis.google.com/js/plusone.js';
-	    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
-	    })();
-	  </script>
+
+        <!-- Twitter -->
+        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+
+        <!-- Google Plus -->
```

Comment:
Maybe it's also time to remove G+?

Reply:
Yes, it is time!
I actually have another iteration of this PR with simple buttons to restore the sharing feature, but without any JS.